packages/shared/prisma/generated/types.ts

```diff
@@ -115,6 +115,17 @@ export const DashboardWidgetChartType = {
     PIVOT_TABLE: "PIVOT_TABLE"
 } as const;
 export type DashboardWidgetChartType = (typeof DashboardWidgetChartType)[keyof typeof DashboardWidgetChartType];
+export const ActionType = {
```

Comment:
I assume this and the migration is fully generated from the schema.prisma. So not checks here. Please run a prisma dev migration to confirm that there is no delta.